core/src/main/scala/org/scalarelational/instruction/Query.scala

```diff
@@ -55,4 +59,43 @@ case class Query[Expressions, Result](expressions: Expressions,
   def async = table.datastore.async {
     result
   }
+
+  def to[R](implicit manifest: Manifest[R]) = {
```

Comment:
We can, but my point was then you wouldn't need `Insert[MappedType]` you would have `Insert` and `MappedInsert[MappedType]`.